src/Files.App/Views/Properties/DetailsPage.xaml

```diff
@@ -91,14 +98,18 @@
 				<ListView.Footer>
 					<Button
 						x:Name="ClearPropertiesButton"
-						Margin="0,4,0,0"
+						Margin="0,8,0,0"
```

Comment:
Can you share a screenshot of the before and after?

Reply:
![image](https://user-images.githubusercontent.com/62196528/232265701-2a257f45-94fb-4a4a-9460-9e1432a92ea9.png)

Reply:
Can you revert this? It's out of scope for this PR and requires some discussion.

Reply:
Reverted